themes/bene_parent/sass/components/_forms.scss

```diff
@@ -102,33 +102,34 @@ input[type='password'],
 input[type='tel'],
 textarea {
   -webkit-appearance: none;
-  background-color: bbv(bene-main-white);
+  background-color: bbv(color-theme-primary-bright);
+  background-color: bbv(--forms-textarea-background-color);
```

Comment:
Variables like this one, with two dashes on the beginning, are accessed with var(...), not bbv(...).